File: app/src/main/java/org/fossasia/susi/ai/rest/responses/susi/ForgotPasswordResponse.java
package org.fossasia.susi.ai.rest.responses.susi;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

/**
 * <h1>POJO class to parse retrofit response from forgot password endpoint susi client.</h1>
 */
public class ForgotPasswordResponse {
    @SerializedName("message")
    @Expose
    private String message;

    /**
     * Gets message.
     *
     * @return the message
     */
    public String getMessage() {
        return message;
    }
}

File: app/src/main/java/org/fossasia/susi/ai/rest/responses/susi/ForgotPasswordResponse.kt
package org.fossasia.susi.ai.rest.responses.susi

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

/**
 * <h1>Kotlin Data class to parse retrofit response from forgot password endpoint susi client.</h1>
 */

data class ForgotPasswordResponse (

    val message: String? = null
)
